python/plugins/processing/algs/gdal/ogr2ogr.py

```diff
@@ -91,6 +97,15 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
 
         arguments.append(output)
         arguments.append(ogrLayer)
-        arguments.append(layerName)
+        if not convertAllLayers:
+            arguments.append(layerName)
 
         return ['ogr2ogr', GdalUtils.escapeAndJoin(arguments)]
+
+    def shortHelpString(self):
+        return self.tr("The algorithm converts simple features data between file formats.\n\n"
+                       "Use convert all layers to convert a whole dataset.\n"
```

Comment:
Thanks for the help string
What do you think about adding the end of this help to the parameter's itself now that they can have their own doc ? See [changelog](https://qgis.org/en/site/forusers/visualchangelog316/index.html#add-help-string-for-parameters) and use at https://github.com/qgis/QGIS/pull/44867/files (_that said, I don't know if it's exposed to Python_)